README.md

```diff
@@ -45,6 +45,9 @@ Follow these steps to kickstart OCP installation on KVM
 On your Terraform client machine:
 1. `git clone https://github.com/ocp-power-automation/ocp4-upi-kvm.git`
 2. `cd ocp4_upi_kvm`
+3. `go get -u github.com/dmacvicar/terraform-provider-libvirt`
+
+   `cp $GOPATH/bin/terraform-provider-libvirt ~/.terraform.d/plugins/darwin_amd64/`
```

Comment:
Here `darwin_amd64` could be `linux_amd64` for linux OS.
Better to copy the plugin binary to repo dir.

Reply:
Please confirm if this looks good -
`cp $GOPATH/bin/terraform-provider-libvirt ocp-power-automation` instead ?

Also, as suggested, I will move this particular piece of documentation to Pre-req Section.